ansible/roles/pretix/templates/nginx.conf.j2

```diff
@@ -17,12 +17,24 @@ server {
   # Security header file not included because frames
 
   location / {
+    if (-f /var/www/pretix/MAINTENANCE_MODE) {
+      error_page 503 =200 /pretix-maintenance.html;
+      return 503;
+    }
+
     proxy_pass http://localhost:8345/;
     proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
     proxy_set_header X-Forwarded-Proto https;
     proxy_set_header Host $http_host;
   }
 
+  location ~ ^/(control|download) { # Available during maintenance mode
```

Comment:
Doesn't this need to be above the / block? Not sure in what order nginx does the handling

Reply:
Nginx matches the longest matching prefix. Order doesn't matter!